Plugin NPC/src/main/java/net/simon987/npcplugin/HarvesterNPC.java

```diff
@@ -36,6 +41,36 @@ public void update() {
         }
     }
 
+    @Override
+    public void setDead(boolean dead) {
+        super.setDead(dead);
+//        GameServer.INSTANCE.getEventDispatcher().dispatch(
+//                new ObjectDeathEvent((Object)this, ID));
+    }
+
+    @Override
+    public void onDeadCallback() {
+        GameServer.INSTANCE.getEventDispatcher().dispatch(
+            new ObjectDeathEvent((Object)this, ID));
+    }
+
+    public BiomassBlob createBiomassBlobInPlace() {
+        Random random = new Random();
+        // random integer in range [2, 4]
+        int yield = random.nextInt(2) + 2;
+
+        BiomassBlob biomassBlob = new BiomassBlob();
+        biomassBlob.setObjectId(GameServer.INSTANCE.getGameUniverse().getNextObjectId());
+        // biomassBlob.setStyle(0); //TODO: set style depending on difficulty level? or random? from config?
+        biomassBlob.setBiomassCount(yield);
+        biomassBlob.setX(getX());
+        biomassBlob.setY(getY());
+        biomassBlob.setWorld(getWorld());
+        LogManager.LOGGER.info("HarvesterNPC spawned biomass in its place");
```

Comment:
Should be LogManager.LOGGER.fine (I realise that you couldn't have known, some contributing guidelines would have been nice on my part).